scripts/biomedical/Human-GEM_Human1/scripts/format_metabolites.py

```diff
@@ -0,0 +1,238 @@
+"""
+This script formats metabolites.tsv, reactantRoles.tsv
+and productRoles.tsv files to
+combine with tMCF files for import to data commons
+"""
+import sys
+import os
+import numpy as np
+import pandas as pd
+import datacommons as dc
+from bioservices import *
+
+
+def main():
```

Comment:
wow, this is one long function. consider breaking it up into helper functions so the code is more readable.